### PR TITLE
[Merged by Bors] - Add documentation to `VisibleEntities` and related

### PR DESCRIPTION
**Pull request by djeedai**
# Objective

Add missing docs

## Solution

Add documentation to the `VisibleEntities` component, its related
`check_visibility()` system, and that system's label.

See Discord discussion here : https://discord.com/channels/691052431525675048/866787577687310356/990432663921901678

### CODE REVIEW

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
Very nice. We should add the current limitation that this has no effect on sprites to these docs, to avoid misleading users until that is fixed.

File: crates/bevy_render/src/view/visibility/mod.rs [hunk 1]
```diff
@@ -49,6 +49,15 @@ impl Default for ComputedVisibility {
 #[derive(Component)]
 pub struct NoFrustumCulling;
 
+/// Collection of entities visible from the current view.
+///
+/// This component contains all app world entities which are visible from the currently
```
Comment (james7132) [inline crates/bevy_render/src/view/visibility/mod.rs:54]:
```suggestion
/// This component contains all app and render world entities which are visible from the currently
```

File: crates/bevy_render/src/view/visibility/mod.rs [hunk 2]
```diff
@@ -49,6 +49,15 @@ impl Default for ComputedVisibility {
...
+/// rendered view. The collection is updated automatically by the [`check_visibility()`]
+/// system, and renderers can use it to optimize rendering of a particular view, to
+/// prevent drawing items not visible from that view.
+///
+/// This component is attached to the same entity as the [`Camera`] and the [`Frustum`]
```
Comment (james7132) [inline crates/bevy_render/src/view/visibility/mod.rs:59]:
```suggestion
/// This component is intended to be attached to the same entity as the [`Camera`] and the [`Frustum`]
```

**Review by james7132 — CHANGES_REQUESTED** [review]

File: crates/bevy_render/src/view/visibility/mod.rs (hunk 1 above)
Comment (djeedai) [reply crates/bevy_render/src/view/visibility/mod.rs:54]: Actually I think it'd be less confusing to just remove "app" here. This is probably not the place to document that app and render worlds are in sync, and phrased as you did @james7132 it sounds to me like the collections contains all entities from the app world AND all entities from the render world (so, twice). Thoughts?

Comment (alice-i-cecile) [reply crates/bevy_render/src/view/visibility/mod.rs:54]:
```suggestion
/// This component contains all entities which are visible from the currently
```

I agree. This is actually clearer and more robust to future changes IMO.

Comment (james7132) [reply crates/bevy_render/src/view/visibility/mod.rs:54]: The Entities of both worlds are kept in sync. An Entity in one is valid in the other.

**Review by alice-i-cecile — APPROVED** [review]

**Review by james7132 — APPROVED** [review]
Please do fix the CI checks. Looks like a doc reference is failing.

**Comment by djeedai** [comment]
I've deleted the link. The clippy issue was the missing `:` between the link and its target (so the link was effectively malformed and ignored). But after that the link becomes broken. I think I had already issues like this in the past, when trying to link to another crate. I can't remember the details, but I think I had pinged the `rustdoc` folks and there was no good solution. Part of the issue is if I remember correctly that the sprite crate is `bevy_sprite` or `bevy::sprite` depending on context, and that confuses `rustdoc`. I can't remember all the details. Anyway, that works now.

Ping @alice-i-cecile for merge, pretty please 😉

File: crates/bevy_render/src/view/visibility/mod.rs [hunk 3]
```diff
@@ -149,6 +163,11 @@ pub fn update_frusta<T: Component + CameraProjection + Send + Sync + 'static>(
     }
 }
 
+/// System updating the visibility of entities each frame.
+///
+/// The system is labelled with [`VisibilitySystems::CheckVisibility`]. Each frame, it updates the
+/// [`ComputedVisibility`] of all entities, and for each view also compute the [`VisibleEntities`]
```
Comment (superdump) [inline crates/bevy_render/src/view/visibility/mod.rs:169]: Too late, but ‘computes’

Comment (djeedai) [reply crates/bevy_render/src/view/visibility/mod.rs:169]: Good catch, sorry I missed that. I'll try to make a new PR when I have some time.

**Pull request closed without merging**